model-optimizer/mo/middle/passes/infer.py

```diff
@@ -108,7 +108,7 @@ def partial_infer(graph: Graph, start_node: str = None):
     graph.strict_mode = True
 
     # Mark all nodes as not inferred yet
-    if not start_node is None:
+    if start_node is not None:
```

Comment:
Why did you change this?

Reply:
This logical expression was in `Joda style`. The last version is easier to understand, in my opinion. How did I get here? I was playing around this file because trying to fix/debug infer of `MemoryOffet` nodes.